src/charm.py

```diff
@@ -206,24 +205,26 @@ def _on_install(self, _) -> None:
             ]
         )
 
-        # mongod.service file passes commandline args to mongod service
-        shutil.copy("src/data/mongod.service", "/etc/systemd/system/mongod.service")
-
-        with open("/etc/systemd/system/mongod.service", "r") as service_template:
+        with open("/lib/systemd/system/mongod.service", "r") as service_template:
             mongodb_service = service_template.readlines()
 
-        mongodb_service[MONGO_EXEC_LINE] = mongod_start_args
+        # replace start command with our parameterized one
+        for i in range(len(mongodb_service)):
```

Comment:
you can use `enumerate` here: 
```suggestion
for index, line in enumerate(mongodb_service):
````